openshift/ci-operator/knative-images/kafka/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Do not edit! This file was generated via Makefile
+FROM registry.svc.ci.openshift.org/openshift/origin-v3.11:base
+
+ADD kafka /usr/bin/kafka
+ENTRYPOINT ["/usr/bin/kafka"]
```

Comment:
Cool. This is a new Dockerfile for the new image. And you also removed building a few "core" binaries so perhaps those Dockerfiles for those images could be removed? I don't see any Dockerfile removed in this PR.

Reply:
something went wrong here ...